### PR TITLE
docs(website): document @feature-hub/dom

### PR DESCRIPTION
**Pull request by fahrradflucht**

### CODE REVIEW

File: docs/guides/writing-a-feature-app.md [hunk 1]
```diff
@@ -226,12 +226,41 @@ const myFeatureAppDefinition = {
...
+If the targeted integrator is using the [`@feature-hub/dom`](dom-api) package a
+Feature App needs to implement the `DomFeatureApp` interface the package
+defines. Since this interface is compatible with the `DomFeatureApp` interface
+defined by [`@feature-hub/react`](react-api) this Feature App will also be
+compatible with an integrator using React.
+
+A DOM Feature App allows fot the use of arbitrary frontend technologies such as
+Vue.js, Angular or React and is [placed on the web page using Web
+Components][placing-feature-apps-on-a-web-page-using-web-components]. The
+Feature App will automatically be enclosed in its own shadow DOM.
```
Comment (KingHenne) [inline docs/guides/writing-a-feature-app.md]: I would add this sentence (from the React section above) as intro to the code snippet:

> Its definition's `create` method returns a Feature App instance with an `attachTo` method that accepts a DOM container element:

File: docs/guides/integrating-the-feature-hub.md [hunk 2]
```diff
@@ -144,6 +144,33 @@ Services][feature-service-dependencies] that are [provided by Feature
...
+The Feature Hub allows for different technology choices for the integrator as
+well as for Feature Apps that the specific integrator supports. While it is
+possbile to build a custom solution tailored to the UI frameworks or libraries
+of choice using the primitives provided by the [`@feature-hub/core`][core-api]
+package, the [`@feature-hub/react`][react-api] and [`@feature-hub/dom`][dom-api]
+package provide out-of-the-box solutions for building an integrator.
+
+The [`@feature-hub/dom`][dom-api] package uses Web Components as a basis. It
+wraps Feature Apps into a shadow DOM and doesn't rely on the presence of any big
+frontend frameworks, which makes it possible to build integrators lean in bundle
```
Comment (KingHenne) [inline docs/guides/integrating-the-feature-hub.md]:
```suggestion
frontend frameworks, which makes it possible to build integrators that are lean in bundle
```

File: docs/guides/writing-a-feature-app.md [hunk 3]
```diff
@@ -226,12 +226,43 @@ const myFeatureAppDefinition = {
...
+Feature App needs to implement the `DomFeatureApp` interface that the package
+defines. Since this interface is compatible with the `DomFeatureApp` interface
+defined by [`@feature-hub/react`](react-api), this Feature App will also be
+compatible with an integrator that uses React.
+
+A DOM Feature App allows for the use of arbitrary frontend technologies such as
+Vue.js, Angular or React, and is [placed on the web page using Web
+Components][placing-feature-apps-on-a-web-page-using-web-components]. The
+Feature App will automatically be enclosed in its own shadow DOM. Its
+definition's create method returns a Feature App instance with an `attachTo`
```
Comment (KingHenne) [inline docs/guides/writing-a-feature-app.md]:
```suggestion
definition's `create` method returns a Feature App instance with an `attachTo`
```

File: docs/guides/integrating-the-feature-hub.md [hunk 4]
```diff
@@ -144,6 +144,33 @@ Services][feature-service-dependencies] that are [provided by Feature
 Apps][own-feature-service-definitions] which are loaded from a remote location,
 instead of being provided by the integrator.
 
+## Choosing an Integrator Technology
+
+The Feature Hub allows for different technology choices for the integrator as
```
Comment (clebert) [inline docs/guides/integrating-the-feature-hub.md]: The Feature Hub provides different technology options for both the integrator and feature apps supported by the integrator.

File: docs/guides/integrating-the-feature-hub.md [hunk 5]
```diff
@@ -144,6 +144,33 @@ Services][feature-service-dependencies] that are [provided by Feature
...
+The [`@feature-hub/dom`][dom-api] package uses Web Components as a basis. It
+wraps Feature Apps into a shadow DOM and doesn't rely on the presence of any big
+frontend frameworks, which makes it possible to build integrators that are lean
+in bundle size. Feature Apps can be build using any technology since they just
+have to render themselves into a provided DOM element.
+
+The [`@feature-hub/react`][react-api] package on the other hand allows for
+building an integrator using React. It has the capability to render Feature Apps
+which are also build using React on the server as well as on the client, while
+it can still integrate Feature Apps that are built using other technologies on
```
Comment (clebert) [inline docs/guides/integrating-the-feature-hub.md]: Maybe we should mention that we have lost SSR as a result?

File: docs/guides/integrating-the-feature-hub.md [hunk 6]
```diff
@@ -317,6 +344,185 @@ When a Feature App throws an error while rendering or, in the case of a
...
+```
+
+#### `instanceConfig`
+
+With the `instanceConfig` property, a config object for a specific Feature App
+instance can be provided:
+
+```js
+const featureAppLoader = document.createElement('feature-app-loader');
+featureAppLoader.setAttribute('src', 'https://example.com/some-feature-app.js');
```
Comment (clebert) [inline docs/guides/integrating-the-feature-hub.md]: newline

File: docs/guides/writing-a-feature-app.md [hunk 7]
```diff
@@ -226,13 +226,45 @@ const myFeatureAppDefinition = {
...
 
+## Implementing a Feature App for an Integrator That Uses Web Components
+
+If the targeted integrator is using the [`@feature-hub/dom`][dom-api] package, a
+Feature App needs to implement the `DomFeatureApp` interface that the package
+defines. Since this interface is compatible with the `DomFeatureApp` interface
+defined by [`@feature-hub/react`][react-api], this Feature App will also be
+compatible with an integrator that uses React.
+
+A DOM Feature App allows for the use of arbitrary frontend technologies such as
```
Comment (clebert) [inline docs/guides/writing-a-feature-app.md]: allows the use of any

File: docs/guides/writing-a-feature-app.md [hunk 8]
```diff
@@ -226,13 +226,45 @@ const myFeatureAppDefinition = {
...
+Vue.js, Angular or React, and is [placed on the web page using Web
```
Comment (clebert) [inline docs/guides/writing-a-feature-app.md]: Angular,

**Pull request merged**